acceptancetests/repository/charms/appdata-sink/config.yaml

```diff
@@ -0,0 +1,5 @@
+options:
+  token:
+    type: string
+    default: ''
```

Comment:
I thought only the source had a token configured, and then that was sent via relation data to the sink

Reply:
I'll remove this.